3Box/3box-profile-hover.md

```diff
@@ -0,0 +1,210 @@
+
+### Summary
+
+
+One of the biggest frictions with onboarding users to Web3 is overcoming a lot of the technical issues that are required by design from decentralized networks and one of them are public addresses which are represented by a 40 character hexadecimal hash which is very unappealing for users.
+
+This issue can be solved using profile hovers which instead of displaying a user's Ethereum address you can display a basic social identity, think of it like using ENS domains but more sociable.
```

Comment:
```suggestion
This issue can be solved using profile hovers which instead of displaying a user's Ethereum address display a basic social identity, think of it like using ENS domains but more sociable.
```

Could you add a link to ENS here? We have an article actually, use that - https://kauri.io/ens:-beginner's-guide/e7f098b57fd24afe9b70506517dd5a63/a